docs/reading.rst

```diff
@@ -56,17 +56,17 @@ Inferring the DataFrame's dtypes
 The :func:`~pandas_gbq.read_gbq` method infers the pandas dtype for each
 column, based on the BigQuery table schema.
 
-================== =========================
+================== ============================================
 BigQuery Data Type dtype
-================== =========================
+================== ============================================
 BOOL               boolean
 INT64              Int64
 FLOAT64            float64
 TIME               dbtime
 DATE               dbdate or object
-DATETIME           datetime64[ns] or object
-TIMESTAMP          datetime64[ns, UTC] or object
-================== =========================
+DATETIME           datetime64[ns] (datetime64[us] if pandas version >= 2.0.0) or object
+TIMESTAMP          datetime64[ns, UTC] (datetime64[us, UTC] if pandas version >= 2.0.0) or object
```

Comment:
Looks like the tests are failing despite the check for 2.0.0. It's possible this behavior was introduced in pandas 2.1.0 because of my PR here: https://github.com/pandas-dev/pandas/pull/52201

```suggestion
DATETIME           datetime64[ns] (datetime64[us] if pandas version >= 2.1.0) or object
TIMESTAMP          datetime64[ns, UTC] (datetime64[us, UTC] if pandas version >= 2.1.0) or object
```

Reply:
Thank you! Sorry for not checking 🙇